test/registry/test-1kg-genotypes-subset.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
can this file be combined with the other `test-genotypes.json` file?  Or you want to keep the partitioning separate?